### ISSUE TITLE
Split cmap-js configurations into several groups

### ISSUE THREAD

**Issue by StevenCannon-USDA** [soybase/jekyll-soybase#11]
The cmap-js view of all QTL features is kind of overwhelming, requiring horizontal scrolling to see all of the feature classes. 

Desired: to divide the features by group, e.g. disease, seed, growth/misc.

**Comment by StevenCannon-USDA** [comment]
I've attempted a solution. See branch "cmap-config":
https://github.com/soybase/jekyll-soybase/tree/cmap-config/maps

Requesting review by at least @jd-campbell and @maxglycine 

Would be nice to reduce the amount of vertical space taken up by the chromosome tables, at maps/index.html . @That-Thing - if you have any ideas for tweaking the layout?

**Comment by That-Thing** [comment]
The vertical space in between the groups could be reduced, or the contents of the groups (ex: 01, 02, etc) could be added to an accordion with the group title as the header.  
An accordion would also allow for a slightly longer description than just a number.

**Comment by StevenCannon-USDA** [comment]
@That-Thing - Could you take this on and see if you can improve the appearance?

**Comment by maxglycine** [comment]
For SoyBase use, I think that the default view should be of the Gm-Composite map only.  The only tracks to display in the default view should be yield, other-seed, protein and oil.  This is because the user could make changes.  They probably do not want to see all the types of QTL, but have a specific type in mind.  Yes, the browser would have to add all the tracks manually.  I think I could live with that.  This way the user has all the options.  We will have to make a video tutorial on how to do this as well as add other maps and link it from the Cmap viewer.

**Comment by StevenCannon-USDA** [comment]
@maxglycine You are saying DON'T provide the other groupings that we discussed yesterday? Disease-related features, for example? Isn't that a convenience for people who are working on those traits?

A video tutorial: some people will use it; most won't. Where a video is required, we're probably doing it wrong. (Not saying that we shouldn't have videos or other guides; just that we should try to make UIs that people can figure out with a bit of experimentation).

**Comment by That-Thing** [comment]
An accordion would look like this:  
![image](https://github.com/soybase/jekyll-soybase/assets/41160238/25e2a1f3-81b3-4201-8d54-8b058a6c707a)

**Comment by StevenCannon-USDA** [comment]
Accordion: I like it. I was unfamiliar with that UI element/method.

**Comment by That-Thing** [comment]
Should I push the change to your branch?

**Comment by StevenCannon-USDA** [comment]
I've made some additional tweaks, now setting the generic "All QTL Features" track-set as "open" by default. Users can open and explore the other track-set if they want - or can compose their own collections of tracks. I think this is in the spirit of your request, @maxglycine.
(@jd-campbell - I added a comment :-) to the code!)
Proposing to close this issue, unless there are other significant suggestions.
<img width="887" alt="map_view" src="https://github.com/soybase/jekyll-soybase/assets/3588740/42c70ebb-3c26-46e1-a8a3-7784d7961499">
